### ISSUE TITLE
Fix color contrast on school page sidebar buttons

### ISSUE THREAD

**Issue by shawnbot** [18F/college-choice#1419]
The school page sidebar buttons have insufficient color contrast (minimum 4.5:1) per WCAG 2.0 AA, as discovered when adding accessibility tests in #1418:

![](https://cloud.githubusercontent.com/assets/113896/12213590/ec2a498c-b62f-11e5-9111-1ef04aa82028.png)

@jjoteal suggests putting the buttons on a white background: 

![](https://cloud.githubusercontent.com/assets/12192706/12273104/fb45ee6c-b931-11e5-8758-3c1218e86aa4.png)

**Comment by jjoteal** [comment]
Hey there,

I don't see this change implemented yet on staging:

![image](https://cloud.githubusercontent.com/assets/12192706/12855583/58651c64-cc0e-11e5-9920-e4c888955e5b.png)


https://federalist.18f.gov/preview/18F/college-choice/staging/school/?164924-Boston-College

That is where I'm supposed to be looking for this change, correct?

**Comment by shawnbot** [comment]
Sorry @jjoteal, let me create a branch with @stevenbuccini's changes so that you can see them on Federalist.

**Comment by shawnbot** [comment]
@jjoteal you can see the proposed fix in #1431 here:

https://federalist.18f.gov/preview/18F/college-choice/sidebar-contrast/school/?123961-University-of-Southern-California

**Comment by jjoteal** [comment]
Looks good! Too bad we can't shorten the text for the first button or make overall grid size adjustments, but as far as this ticket goes, this is ready to roll!

**Comment by shawnbot** [comment]
Okay, great, thanks @jjoteal!

**Comment by shawnbot** [comment]
Reopening for confirmation in the next deployment.

**Comment by barkimedes** [comment]
:us: looks good!
